tests/e2e/core-tests/specs/merchant/wp-admin-product-import-csv.test.js

```diff
@@ -63,7 +63,7 @@ const runImportProductsTest = () => {
 			await expect(page).toClick('button[value="Run the importer"]');
 
 			// Waiting for importer to finish
-			await page.waitForSelector('section.woocommerce-importer-done', {visible:true, timeout: 60000});
+			await page.waitForSelector('section.woocommerce-importer-done', {visible:true, timeout: 120000});
```

Comment:
Nothing we need to change, just came to mind as I was going through this. I wonder if we can improve on this by:

* Adding a check for `spinner.is-active` which is the loading icon the page with the loading bar:

<img width="822" alt="Screen Shot 2021-05-03 at 1 20 50 PM" src="https://user-images.githubusercontent.com/71906536/116923330-bc64a280-ac13-11eb-9968-20c19e2b3966.png">

* Checking for a page navigation from the load bar during the import process (the page reloads just before going to the success screen). 

I am not sure if either of the above would help, or if just adding in the higher timeout might be the safest.

Reply:
I'm not sure that would help here since this timeout is a maximum limit vs being a delay. But at some point we should look at all the small hard coded delays we've added to see if there is something on the page we can use instead.

Reply:
That makes sense.

> But at some point we should look at all the small hard coded delays we've added to see if there is something on the page we can use instead.

I like that idea. I added this issue for that: https://github.com/woocommerce/woocommerce/issues/29818